src/sctools/platform.py

```diff
@@ -629,3 +629,313 @@ def attach_barcodes(cls, args=None):
         cls._tag_bamfile(args.u2, args.output_bamfile, tag_generators)
 
         return 0
+
+
+class BarcodePlatform(GenericPlatform):
+    """Command Line Interface for extracting and attaching barcodes with specified positions
```

Comment:
For an assay description: I was concerned since the class has been designed for a more generic use case, providing a particular assay as an example might confuse the user

Reply:
Got it, I was confused. See comment below!